### PR TITLE
add development/configuration.rst

### PR DESCRIPTION
**Pull request by hotta**

### CODE REVIEW

File: ja/development/configuration.rst [hunk 1]
```diff
@@ -1,16 +1,853 @@
...
+                'renderer' => 'ExceptionRenderer',
+                'log' => true
+            )
+        );
+
+    ファイルに ``$config`` がなかった場合は :php:exc:`ConfigureException`
+    が発生します。
+
+    あなたのカスタムの設定ファイルをロードするには、
+    app/Config/bootstrap.php に以下のコードを挿入します:
```
Comment (markstory) [inline ja/development/configuration.rst]: Should be `::`

**Comment by antograssiot** [comment]
Formatting looks good now.
thanks @hotta  :+1:

**Comment by okinaka** [comment]
:+1

**Comment by okinaka** [comment]
:+1:

**Pull request merged**